src/components/generic/movieModal/movieModal.tsx

```diff
@@ -1,32 +1,10 @@
 import * as React from 'react'
 import {FC} from 'react'
 
-import ModalContent from './ModalContent'
+import {useModal} from '@/context/modal-context'
+import {useMovie} from '@/api/movie_api'
```

Comment:
You really have to add a linter that detects unused variables!!!

Reply:
I do, but its not showing for all files